modules/appeals_api/spec/support/doc_helpers.rb

```diff
@@ -65,7 +65,7 @@ def self.decision_reviews_security_config
   end
 
   # @param [Hash] opts
-  # @option opts [String] :cassette The name of the cassette to use, if any
+  # @option opts [String|String[]] :cassette The name(s) of the cassette(s) to use, if any
```

Comment:
I hadn't yet had to use multiple cassettes in a single rswag example, but these changes support that.